src/transformers/configuration_utils.py

```diff
@@ -163,7 +163,7 @@ class PretrainedConfig(object):
 
     def __init__(self, **kwargs):
         # Attributes with defaults
-        self.return_dict = kwargs.pop("return_dict", False)
+        self.return_dict = kwargs.pop("return_dict", True)
```

Comment:
this should not be shown in this PR here I think -> isn't that already changed? Maybe rebase to master?

Reply:
Yes I know. I did this to reflect the current API of Transformers (v4). I'm going to open a clean PR with all the added files. This branch is out-of-date and a rebase would be kind of painful.